src/ai/backend/test/contexts/client_session.py

```diff
@@ -1,8 +1,14 @@
-from ai.backend.client.session import AsyncSession
+from ai.backend.client.session import AsyncSession, Session
 from ai.backend.test.contexts.context import BaseTestContext, ContextName
 
 
 class ClientSessionContext(BaseTestContext[AsyncSession]):
     @classmethod
     def name(cls) -> ContextName:
         return ContextName.CLIENT_SESSION
+
+
+class ClientSyncSessionContext(BaseTestContext[Session]):
+    @classmethod
+    def name(cls) -> ContextName:
+        return ContextName.CLIENT_SESSION
```

Comment:
Wouldn't it be okay to just receive both cases as BaseSession? (In that case, it seems like async and sync can both be unwrapped from the top.)